packages/wdio-logger/tests/node.test.js

```diff
@@ -1,5 +1,6 @@
 import fs from 'fs'
 import nodeLogger from '../src/node'
+import nodeLogger2 from '../src/nodeCopy'
```

Comment:
Hi @mgrybyk, sorry that it's not clear what I was asking you. I'm asking if you can think of _a better way_ to capture the bug in a failing test. Because I'd like to have a test that fails because it's capturing the bug before having a bug fix to make it pass.

I have a way to capture the bug using our existing tests (hence why they are failing), but I don't think it's the best way. What I am doing is importing a second copy of `src/node.js` in the test. This is my way of reproducing the bug. This bug is only reproducable when we run multiple copies of the same source code, as happens when multiple wdio packages declare wdio-logger as a dependency.

I've updated the PR description to explain this better.